docs/src/resources/troubleshooting.md

```diff
@@ -1,5 +1,101 @@
 # Troubleshooting
 
-This guide helps you troubleshoot common issues with Runwasi.
+This guide helps you troubleshoot common issues with runwasi shims.
 
-[TODO]
+## Container Fails to Start
+
+If your wasm container fails to start, you might see events like these:
+
+```
+Events:
+  Type     Reason                  Age               From               Message
+  ----     ------                  ----              ----               -------
+  Normal   Scheduled               18s               default-scheduler  Successfully assigned default/wasi-demo-75d6bb666c-479km to minikube
+  Warning  FailedCreatePodSandBox  4s (x2 over 18s)  kubelet            Failed to create pod sandbox: rpc error: code = Unknown desc = failed to get sandbox runtime: no runtime for "wasmtime" is configured
+```
+
+#### Investigation Steps:
+
+1. **Check runtime class configuration**: Ensure your `RuntimeClass` is correctly configured.
+   ```bash
+   kubectl get runtimeclass -o yaml
+   ```
```

Comment:
What should it look like? Maybe mention the link between this and the containerd configuration.